### DISCUSSION TITLE
Needs for two features

### DISCUSSION THREAD

**Discussion by NailGit** [ahrm/sioyek#817] [Ideas]
I have used many years mupdf and just found Sioyek which is a very good program. There are not enough two functions for scientific reading of pdf's. 1) Two-page functionality 2) Save highlights directly in the file. The last one is useful to send somebody a file with my notes. It has to be open in any pdf viewer with visible highlights.

**Comment by ahrm** [comment]
There is an `embed_annotations` command that creates a new PDF file with highlights embedded.

**Comment by NailGit** [comment]
Oh yes, thanks. What about a two-page functionality?  I saw discussions about this in the forum. I know that such an opportunity is not provided in mupdf and realization of this possibility will be a very difficult problem.

**Reply by ahrm** [reply]
I may implement it in the future, but I have no immediate plans for that.

**Reply by NailGit** [reply]
Thank you in advance.
